project_report/tex/introduction.tex

```diff
@@ -96,3 +96,51 @@ \subsubsection{REST API}
 frontend to receive data via fetch calls in JavaScript, it needs to communicate 
 with something that uses HTTP and preferably delivers data in JSON.
 
+
+\subsection{Testing the solution}
+To validate the developed system, the group uses the brewery machine and the
+brewery simulator provided by SDU. A short description of the two will be given
+in the following subsections, including how the group uses them to run tests.
+
+\subsubsection{The Physical Setup (The Brewery Machine)}
+The machine can produce different kinds of beer, depending on the recipe used.
+The beer is produced in batches which have their own batch id. When producing a
+batch, the recipe, quantity, speed and batch id can be configured by an external
+system. When a batch has been produced, the quality inspection system measures
```

Comment:
Very short 'boxed' sentences. Consider rewriting at least one of them, to a longer sentence with a comma instead of a period.